test/corpus/expressions

```diff
@@ -39,6 +39,34 @@ f(1) + f(1);
               (integer)))))))
   (nl))
 
+================================================================================
+IDs
+================================================================================
+
+a;
+a01231;
+a::b;
+Cluster::Supervisor::__init_cluster_nodes();
```

Comment:
nit: any way a global ID could be added to the test (with leading `::`)?

Reply:
Added a test for that and also the legacy syntax with `GLOBAL::...`.